specs/tilebag_spec.rb

```diff
@@ -0,0 +1,76 @@
+require 'pry'
+require 'minitest/autorun'
+require 'minitest/reporters'
+require 'minitest/skip_dsl'
+
+require_relative '../lib/tilebag'
+
+Minitest::Reporters.use! Minitest::Reporters::SpecReporter.new
+
+describe 'TileBag' do
+
+  describe 'initialize' do
+
+    it "Sets up instance with a collection of tiles" do
+      new_pool = Scrabble::TileBag.new
+      quantities = { "a" => 9,"b" => 2,"c" => 2,"d" => 4,"e" => 12,"f" => 2,"g" => 3,"h" => 2, "i" => 9,"j" => 1,"k" => 1, "l" => 4,"m" => 2,"n" => 6,"o" => 8,"p" => 2, "q" => 1, "r" => 6, "s" => 4, "t" => 6, "u" => 4, "v" => 2, "w" => 2, "x" => 1, "y" => 2, "z" => 1 }
+      new_pool.total_tiles.must_equal quantities
```

Comment:
I don't know that I like this test - seems like you're just copying the data that you have in tile_bag.rb. That means it's not likely to find a bug, and if that data ever changes (Scrabble rules change or you do find a bug) you'll have to make the update in more than one place.

Instead, it might be better in this case to test _less_. Maybe check that you get the right number of tiles and leave it at that.